packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -9,13 +9,65 @@ import { DisabledMixin } from '@vaadin/a11y-base/src/disabled-mixin.js';
 import { defineCustomElement } from '@vaadin/component-base/src/define.js';
 import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { TooltipController } from '@vaadin/component-base/src/tooltip-controller.js';
 import { matchPaths } from '@vaadin/component-base/src/url-utils.js';
 import { LumoInjectionMixin } from '@vaadin/vaadin-themable-mixin/lumo-injection-mixin.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { location } from './location.js';
 import { sideNavItemStyles } from './styles/vaadin-side-nav-item-base-styles.js';
 import { SideNavChildrenMixin } from './vaadin-side-nav-children-mixin.js';
 
+/**
+ * A controller for observing tooltip slotted in the item.
+ * @private
+ */
+export class ItemTooltipController extends TooltipController {
```

Comment:
Preferrably we would also hide the tooltip element itself from screen readers since we duplicate its contents into a separate element. So maybe set `aria-hidden` on it when it is slotted into a side nav item.